Serenity-Website/src/MoodBoard/MoodBoard.jsx

```diff
@@ -1,12 +1,127 @@
-import './MoodBoard.css'
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCalendar, faX } from "@fortawesome/free-solid-svg-icons";
+
+import './MoodBoard.css';
+import angry  from '../assets/angry.png';
+import sad from '../assets/sad.png';
+import neutral from '../assets/neutral.png';
+import happy from '../assets/happy.png';
+import excited from '../assets/excited.png';
+import Header from "../Header/Header";
+
+const moods = {
+    // moood object 
+    angry: <img src={angry}/>,
+    sad: <img src={sad} />,
+    neutral: <img src={neutral}/>,
+    happy: <img src={happy}/>,
+    excited: <img src={excited} />
+
+};
 
 function MoodBoard (){
+    // set default mood to null
+    const [selectedMood, setSelectedMood] = useState(null);
+    
+    //set current date
+    const [currentDate, setCurrentDate] = useState('');
+    
+    //state hook to set success message for when user submits a mood.
+    const [successMessage, setSuccessMessage] = useState('');
+    
+    //track if mood has been submitted
+    const [moodSubmitted, setMoodSubmitted] = useState(false);
+
+    // set error message
+    const[errorMessage, setErrorMessage] = useState('');
+
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        // Set the current date and render when the component mounts 
```

Comment:
The way you are currently updating the currentDate, you'll only update it once per user session. 

Let's say I open your app at 3:21 PM on July 8th. If you go to record a mood at 3:30PM, it'll still say it's 3:21PM. This doesn't seem like your intended behavior.

Reply:
My question is, do you need to track this Date object? I don't think displaying date is necessary for this page.

Reply:
Fixed this! Now displays only the day and date not time.